nautobot/circuits/migrations/0017_fixup_null_statuses.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+
+from nautobot.extras.utils import fixup_null_statuses
+
+
+def migrate_null_statuses(apps, schema):
+    status_model = apps.get_model("extras", "Status")
+    ContentType = apps.get_model("contenttypes", "ContentType")
+    for model_name in ("Circuit",):
```

Comment:
Is the for loop necessary here?

Reply:
No, just laziness that I could try to justify as self-consistency. :-)